UseCases/Sources/ManualContextPropagation/main.swift

```diff
@@ -105,3 +102,33 @@ private struct FakeTracer {
         static let headerName = "fake-trace-id"
     }
 }
+
+private struct SecondFakeTracer {
```

Comment:
maybe no need to showcase these in the samples? will add noise and we know how the multiplex one works from the tests (and docs)